package/boot/arm-trusted-firmware-mvebu/Makefile

```diff
@@ -0,0 +1,112 @@
+#
+# Copyright (C) 2019 OpenWrt
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=arm-trusted-firmware-mvebu
+PKG_RELEASE:=2.1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_RELEASE).tar.xz
+PKG_SOURCE_URL=https://github.com/ARM-software/arm-trusted-firmware
```

Comment:
I would prefer main upstream repo (https://git.trustedfirmware.org/TF-A/trusted-firmware-a.git) instead of mirror.

Reply:
@tmn505 I've update PKG_SOURCE_URL to use upstream as suggested.